test/functional/signrawtransactions.py

```diff
@@ -33,11 +34,19 @@ def successful_signing_test(self):
         outputs = {'mpLQjfK79b7CCV4VMJWEWAj5Mpx8Up5zxB': 0.1}
 
         rawTx = self.nodes[0].createrawtransaction(inputs, outputs)
+        rawTxSigned = self.nodes[0].signrawtransactionwithkey(rawTx, privKeys, inputs)
```

Comment:
I don't think you need to repeat all these tests twice. Just add something like the following to the end of each section:

```python
rawTxSignedWithKey = self.nodes[0].signrawtransactionwithkey(rawTx, privKeys, inputs)
assert_equal(rawTxSigned, rawTxSignedWithKey)
```

Reply:
Done